harvest/trader/trader.py

```diff
@@ -89,7 +89,7 @@ def _init_attributes(self):
         self.logger = BaseLogger()
         self.server = Server(self)  # Initialize the web interface server
 
-        self.timezone = tzlocal.get_localzone()
+        self.timezone = tzlocal.get_localzone_name()
```

Comment:
get_localzone is deprecated, use get_localzone_name instead

Reply:
I think this causes a crash whenever timezone is converted from UTC to local or vice versa, since many functions assume `self.timezone` to be a zoneinfo object

Reply:
> I think this causes a crash whenever timezone is converted from UTC to local or vice versa, since many functions assume `self.timezone` to be a zoneinfo object